DependencyInjection/SonataAdminExtension.php

```diff
@@ -111,8 +111,7 @@ public function load(array $configs, ContainerBuilder $container)
                         'VIEW' => array('VIEW'),
                         'DELETE' => array('DELETE'),
                         'EXPORT' => array('EXPORT'),
-                        'OPERATOR' => array('OPERATOR'),
-                        'MASTER' => array('MASTER'),
+                        'ALL' => array('ALL'),
```

Comment:
Yes. The key represents the role you will see on the dashboard, and the array represents which permissions you are getting.

On the acl section there are some cases where one role grants you more than one permission. It is not happening on "role based security" that's why it looks strange.